components/infra/docker-compose.yml

```
@@ -6,11 +8,13 @@ services:
       - .env
     restart: always
     healthcheck:
-      test: echo 'db.runCommand("ping").ok'
+      test: [ "CMD-SHELL", "mongosh --port ${MONGO_PORT} --eval 'db.adminCommand({ping:1})' --quiet" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)